core/MySigning.cpp

```diff
@@ -200,10 +206,11 @@ bool signerSignMsg(MyMessage &msg)
 			_signingNonceStatus=SIGN_WAITING_FOR_NONCE;
 			if (!_sendRoute(build(_msgSign, msg.destination, msg.sensor, C_INTERNAL,
 			                      I_NONCE_REQUEST).set(""))) {
-				SIGN_DEBUG(PSTR("Failed to transmit nonce request!\n"));
+				SIGN_DEBUG(PSTR("!SGN:SGN:NCE REQ TO=%d\n"),
+				           msg.destination); //Failed to transmit nonce request!
 				ret = false;
 			} else {
-				SIGN_DEBUG(PSTR("Nonce requested from %d. Waiting...\n"), msg.destination);
+				SIGN_DEBUG(PSTR("SGN:SGN:NCE REQ TO=%d\n"), msg.destination); //Nonce requested
```

Comment:
There are no "law" as such. I think, the most important thing is to keep code and documentation in line. But I am updating the parts where there is "SOMETHING TO=%d" or "SOMETHING FROM=%d" to SOMETHIING,TO=%d" etc. as that seem to be the most commonly used formatting. Spaces in other parts of the last entry (MOD:SUB:MSG) seem to be "permitted". And I think it is a case-by-case thing where clarity wins.